packages/plugins/typescript/nhost/tests/nhost.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { buildSchema } from 'graphql';
+import { plugin } from '../src/index.js';
+
+describe('TypeScript Operations Plugin', () => {
```

Comment:
```suggestion
describe('TypeScript Nhost Plugin', () => {
```

Reply:
Or this is extra in general?

Reply:
no, your suggestion makes sense

Reply:
Then this is extra;
https://github.com/dotansimha/graphql-code-generator-community/pull/269/files#diff-980d5a880ad6793f2ff5f0f02eeb168103ea4c72f7d082c427c3e3e288b467b7R87

Reply:
corrected